src/TokenType.ts

```diff
@@ -0,0 +1,475 @@
+export interface BackgroundColorType {
```

Comment:
NIT: I think it would make more sense to call this file `TokenTypes.ts`

Reply:
Done